includes/Classifai/Admin/SavePostHandler.php

```diff
@@ -248,4 +271,36 @@ public function is_rest_route() {
 		return false;
 	}
 
+	/**
+	 * Classify post manually.
+	 *
+	 * @return void
+	 */
+	public function classifai_classify_post() {
+		if ( ! empty( $_GET['classifai_classify_post_nonce'] ) && wp_verify_nonce( sanitize_text_field( $_GET['classifai_classify_post_nonce'] ), 'classifai_classify_post_action' ) ) {
+			$post_id = isset( $_GET['post_id'] ) ? absint( $_GET['post_id'] ) : 0;
+			if ( $post_id ) {
+				$result     = $this->classify( $post_id );
+				$classified = array();
+				if ( ! is_wp_error( $result ) ) {
+					$classified = array( 'classifai_classify' => 1 );
+				}
+				wp_safe_redirect( esc_url_raw( add_query_arg( $classified, get_edit_post_link( $post_id, 'edit' ) ) ) );
+				exit();
+			}
+		} else {
+			die( esc_html__( 'You don\'t have access to doing this operations.', 'classifai' ) );
```

Comment:
```suggestion
			wp_die( esc_html__( 'You don\'t have permission to perform this operation.', 'classifai' ) );
```

`wp_die()` is a little nicer looking than `die()`